docs/changelog_de.md

```diff
@@ -15,6 +15,7 @@
 * Fix [#129](https://g1cp.org/issues/129): Drakes Hautfarbe des Körpers entspricht jetzt der seines Kopfes.
 * Fix [#220](https://g1cp.org/issues/220): Gor Na Ran no longer attacks the player character in chapter 6.
 * Fix [#224](https://g1cp.org/issues/224): Grash-Varrag-Arushat (der letzte untote Ork-Priester) kann nicht mehr durch Fallschaden getötet werden.
+* Fix [#225](https://g1cp.org/issues/225): Mordrag kann nicht mehr gesagt werden "Mach, dass du wegkommst", wenn er den Spieler-Charakter bereits ins Neue Lager geführt hat.
```

Comment:
I would suggest to just write "Spieler" here, not "Spieler-Character", since the player themself is actually guided. In the English version of this changelog entry it is also just "player".